Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocValidationErrorIndicatorColumnRenderer.cs

```diff
@@ -0,0 +1,100 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+using Remotion.ServiceLocation;
+using Remotion.Utilities;
+using Remotion.Web;
+using Remotion.Web.Infrastructure;
+using Remotion.Web.UI.Controls;
+using Remotion.Web.UI.Controls.Rendering;
+
+namespace Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation.Rendering
+{
+  [ImplementationFor(typeof(IBocValidationErrorIndicatorColumnRenderer), Lifetime = LifetimeKind.Singleton)]
+  public class BocValidationErrorIndicatorColumnRenderer : BocColumnRendererBase<BocValidationErrorIndicatorColumnDefinition>, IBocValidationErrorIndicatorColumnRenderer
+  {
+    private const string c_nonBreakingSpace = " ";
```

Comment:
```suggestion
    private const string c_whiteSpace = "&nbsp;";
```
I usually use this pattern. We can discuss the lack of naming specifity but there's already a couple and we should keep it consistent and maybe change it in one swoop.